Makefile

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2022 Toitware ApS.
+# Use of this source code is governed by a Zero-Clause BSD license that can
+# be found in the tests/LICENSE file.
```

Comment:
It is a little bit weird that we use a license from a subdirectory for these, but it's probably fine.

Reply:
yeah. That was by accident. But I'm happy if someone just wants to copy it.